traitsui/table_filter.py

```diff
@@ -518,6 +517,7 @@ def _get_table_editor(self, names):
         """ Returns a table editor to use for editing the filter.
         """
         from .api import TableEditor
+        from .editors.api import EnumEditor
```

Comment:
nit: maybe make these not relative imports?
That could be done in a separate PR if need be as well

Reply:
done.